src/AnkiService.ts

```diff
@@ -142,6 +142,17 @@ export class AnkiService {
     return await this.invoke("addNotes", { notes });
   }
 
+  async resetUi(): Promise<any[]> {
+    return await this.invoke("guiDeckBrowser");
+  }
```

Comment:
Whats the reason for this?
Also we should keep the method names similar to the API names, so that if others need the service it's easy to know what matches up with what.